### DISCUSSION TITLE
Change date and remove reading time

### DISCUSSION THREAD

**Discussion by Gzrayyy** [chrede88/qubt#89] [Q&A] [answered]
Hi there,

I really love this theme! Right now, I'm building my blog based on the template on RStudio. 

Here is an issue. When I added publication content, I found that the date was wrong (always Jan 11), and I tried to copy the code that you wrote and change the time manually, but it didn't really work (only worked for some specific dates; otherwise,  it is still Jan 1-1). 

Another question is: how can I remove the reading time?

![75bcba62a63003bceca4353109a2228](https://github.com/chrede88/qubt/assets/58963327/30bcea9c-8987-41c8-b5a7-a0ee990109da)
![ce26e5dae901c7ab3ff59550b92971a](https://github.com/chrede88/qubt/assets/58963327/da6496bb-c2af-439f-9ba5-092fc1787ee8)

**Comment by chrede88** [comment, marked as answer]
**Date:**
Define in the frontmatter: `date: 2023-10-19`
Like I've done here for the example site: https://raw.githubusercontent.com/chrede88/qubt/main/exampleSite/content/blog/post1/index.md

**Reading Time:**
If you don't want the reading time to be displayed you've to modify the partial. This will overwrite it for everything. So you won't see reading time anywhere. If this is not what you want, let me know.
Overwrite this file: https://github.com/chrede88/qubt/blob/main/layouts/_default/single.html#L5
In your own repo add this to a new file under `./layouts/_default/single.html` and change line 5 to this:
```
{{ $DPT := $date }}
```

~~I've created a test repo that should do what you want: https://github.com/chrede88/quit-pub~~
